main_test.py

```diff
@@ -68,3 +115,27 @@
     logger((e,), channel='error')
     raise e
 
+
+
+
+
+
+
+# async def a():
+#     from libs.controllers.network import Frame
+
+#     orig_time = node.timekeeping_controller.get_time()
+#     f = Frame(
+#         Frame.FRAME_TYPES['sync_time'],
+#         (orig_time - 10000000).to_bytes(4, 'big'),
+#         nc.address,
+#         1
+#     )
+#     node.network_controller.on_message(f.serialize())
+
+#     new_time = node.timekeeping_controller.get_time()
+
+#     print(orig_time, new_time, orig_time - new_time)
+
+
+
```

Comment:
I am fairly sure this was at some point deleted on the main branch. please remove it again.